### PR TITLE
add cloud 0105 release node

### PR DESCRIPTION
**Pull request by wingwing2005**

### CODE REVIEW

File: docs/en/20-releases/05-20240105.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+---
+
+TDengine Cloud released 20240105 on January 5th, 2024. Please see the following details.
+
+## Features
+
+1. Support the specified database for favorite SQL.
+2. Add data-in resources checking before suspending and deleting a instance.
+3. Deactivate console.cloud.tdengine.com URL and please access cloud.tdengine.com.
+4. Support downloading the related Taos tool based on the TDengine version.
```
Comment (danielclow) [inline docs/en/20-releases/05-20240105.md]: Is it taostools or TDengine client?

Comment (wingwing2005) [reply docs/en/20-releases/05-20240105.md]: TDengine client

Comment (wingwing2005) [reply docs/en/20-releases/05-20240105.md]: We can change it to "TDengine client"

Comment (wingwing2005) [reply docs/en/20-releases/05-20240105.md]: @danielclow have updated.

**Pull request merged**